src/ui/components/Explorer.js

```diff
@@ -316,7 +312,7 @@ export class Explorer extends React.Component<ExplorerProps, ExplorerState> {
           <TableHead>
             <TableRow>
               <TableCell align="left" style={{color: "black"}}>
-                Node
+                {name ? name : "All users"}
```

Comment:
Nit: Would it be possible to prefix the filter group (like the plugin name here, so if you select "Pull requests" from the dropdown you'll see "Github: Pull Requests"

Reply:
This is possible but probably a bigger pain then what its worth. (Adding a bunch of state logic and splitting strings)